ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -579,33 +469,376 @@ def click_actions_control_then_start(self):
+        Returns:
+            str: The LUN group name that was deleted.
+        """
+        logger.info("Fetching LUN group name")
+        ocp = OCP(kind="filesystem", namespace="ibm-spectrum-scale")
+        fs_out = ocp.exec_oc_cmd(
+            "get filesystem -n ibm-spectrum-scale --no-headers",
+            out_yaml_format=False,
+        )
+        lungroup_name = None
+        for line in fs_out.splitlines():
+            line = line.strip()
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)